web/core/components/views/form.tsx

```diff
@@ -194,7 +199,7 @@ export const ProjectViewForm: React.FC<Props> = observer((props) => {
                     hasError={Boolean(errors.name)}
                     placeholder="Title"
                     className="w-full text-base"
-                    tabIndex={1}
+                    tabIndex={getTabIndex("name", ETabIndices.PROJECT_VIEW, isMobile)}
```

Comment:
Can you curry the getTabIndex function becuase the isMobile argument is redundant?

Reply:
Resolved